hr_attendance_report_theoretical_time/views/hr_employee_views.xml

```diff
@@ -10,7 +10,7 @@
         <field name="view_mode">form</field>
         <field
             name="context"
-        >{'default_employee_ids': active_ids, 'employee_ids': active_ids, }</field>
+        >{'default_employee_ids': [id], 'employee_ids': [id] }</field>
```

Comment:
Please check this. If I try to use this action from an employee there is an error.

Reply:
I can't find any errors.
If you test it again, please include a screenshot and I'll look into whether I can fix it. Thanks Victor

Reply:
![ejemplo](https://github.com/user-attachments/assets/d3a8b198-92cd-46a8-891b-d9f8a44fd272)

Reply:
Thank you, I was only able to get it to work by reverting to using active_ids, even though it will be deprecated in Odoo 18. Right now, [id] or [(4, id)] don't work.